docs/data/frontier-squid.md

```diff
@@ -167,8 +167,15 @@ To register your Frontier Squid host, follow the general registration instructio
 2.  If you are setting up a new resource, set `Active: false`.
     Only set `Active: true` for a resource when it is accepting requests and ready for production.
 
-If you are running a WLCG site, a few hours after a squid is registered
-and marked `Active`, 
+3.  By default registered squids will be monitored by WLCG.  This is
+strongly recommended even for non-WLCG sites so operations experts can
+help with diagnosing problems.  However, if a site declines
+monitoring, that can be indicated by adding `Monitored: false`
+directly below `ID`.  Registration is still important for the sake of
```

Comment:
I don't think "directly below `ID`" is correct. What they need to do is add a `Details:` section into the `Squid:` section, and inside the `Details:` section, add `Monitored: false`.

Since that wording is complicated, an example would be helpful. Perhaps it's easier to replace the example in step 1 with:
```
FQDN: <FULLY QUALIFIED DOMAIN NAME>
Services:
  Squid:
    Description: Generic squid service
    Details:
      Monitored: true
...
```
...and tell them to change `Monitored: true` to `Monitored: false` if they don't want monitoring.

Reply:
Oh right, sorry, I didn't pay enough attention to your original message about how to do it.  I will revise it.

Reply:
@DrDaveD I'd prefer if we also added it to the markdown example

Reply:
The reason why I’d rather not include it in the markdown example is that I want to discourage disabling the monitoring, and that people don’t normally need to set it if it is true.  People need to look at the complete example in order to really create one of these entries anyway, so I think that's enough.

Reply:
In that case, I don't think we should obfuscate configuration but rather provide solid reasons why admins shouldn't disable it.

Reply:
There is already a solid reason given: "This is strongly recommended even for non-WLCG sites so operations experts can help with diagnosing problems".  So in almost all cases including it in the example snippet will just be leading people to go against that recommendation.   I would rather that they ask the experts how to do it, so the experts have a chance to try to talk them out of it.

Reply:
@brianhlin Can we resolve this?

Reply:
@brianhlin This is still an open pull request since January, can we go ahead with this?